src/index.html

```diff
@@ -176,6 +176,9 @@ <h2 id="contributing">Contributing</h2>
         <h1>Made with love.</h1>
         <p>By <a href="https://twitter.com/wooorm">@wooorm</a>.</p>
         <p><small>&hellip;and a special thanks to <a href="https://twitter.com/sindresorhus">@sindresorhus</a> for inspiring me to do this.</small></p>
+        <br/>
+        <p>Contribute to Alex for the GitHub Social Impact Collection Challenge during the month of October for Hacktoberfest to receive a limited edition badge on your GitHub Community Forum profile.  For more information, please click the image below.</p>
```

Comment:
Maybe a link here? https://github.com/collections/social-impact

Reply:
Maybe both?!

Hey, we could also add this more at the start of the page? This is very low!

Reply:
how does this look? 
![hacktoberfest_screenshot2](https://user-images.githubusercontent.com/10003710/46443536-fb6ca780-c73b-11e8-9a9e-d9a17e2119c6.png)

Reply:
or put the text under the image? 
![hacktoberfest_screenshot3](https://user-images.githubusercontent.com/10003710/46443649-6e761e00-c73c-11e8-9467-a54ed6b1a8f4.PNG)

Reply:
I like the last one, image first, text second.

* If you still need some spacing, maybe a heading makes more sense semantically than a `<br>` element. Or some CSS?
* For the link in the text, could you make it a bit more descriptive? `click here` is not so good for [accessibility or SEO](http://stephanieleary.com/2015/05/why-click-here-is-a-terrible-link-and-what-to-write-instead/)
* And could you remove `target="_blank"` from the link? It’s fine if the link loads in the original tab/window, I think, but maybe you had a reason for it!